centreon/www/class/centreonLDAP.class.php

```diff
@@ -455,7 +455,7 @@ public function listOfUsers($pattern = '*'): array
      * @param array $attr The list of attribute
      * @return array|bool The list of information, or false in error
      */
-    public function getEntry($dn, $attr = [])
+    public function getEntry($dn, $attr = []): array|bool
```

Comment:
In PHP 8.2 you can type with true or false

```suggestion
    * @return array|false The list of information, or false in error
    */
    public function getEntry($dn, $attr = []): array|false
```